sonar-scanner-engine/src/main/java/org/sonar/scanner/bootstrap/GlobalContainer.java

```diff
@@ -78,7 +80,8 @@ private void addBootstrapComponents() {
       BatchPluginPredicate.class,
       ExtensionInstaller.class,
 
-      SonarRuntimeFactory.create(System2.INSTANCE, SonarProduct.SONARQUBE, SonarQubeSide.SCANNER),
+      new SonarQubeVersion(apiVersion),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "SonarQubeVersion"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)